content/sensu-go/5.0/installation/plugins.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Installing Sensu Plugins"
+linkTitle: "Install Plugins"
+description: "The Sensu Core installation guide."
+weight: 2
+version: "5.0"
+product: "Sensu Go"
+menu:
+  sensu-go-5.0:
+    parent: installation
+---
+
+Sensu's functionality can be extended through the use of plugins.
+Plugins can provide executables for performing status or metric checks, mutators for changing data to a desired format, or handlers for performing an action on a Sensu event.
+You can find a number of plugins in the [Sensu Plugins][1] organization on GitHub.
+
+_NOTE: Plugins found in the Sensu Plugins GitHub organization are community-maintained, meaning that anyone can improve on a plugin found there. If you have a question about how you can get involved in adding to, or providing a plugin, head to the [Sensu Community Slack channel][4]. Maintainers are always happy to help answer questions and point you in the right direction._
+
+### Installing Sensu Plugins
+To install a Sensu Community Plugin with Sensu Go:
+
+1. Install the [sensu-plugins-ruby package][2] from packagecloud.
+
+2. Use the `sensu-install` command to install any plugins in the [Sensu Plugins organization on GitHub][1] by repository name. Plugins are installed into `/opt/sensu-plugins-ruby/embedded/bin`.
```

Comment:
Will probably need to run sensu-install  via sudo, unless user running sensu-install has permissions to write to /opt/sensu-plugins-ruby/.  By default this generally isnt the case so sudo is commonly needed.